apps/src/netsim/netsim.js

```diff
@@ -507,8 +513,10 @@ NetSim.prototype.disconnectFromShard = function (onComplete) {
   this.myNode.stopSimulation();
   this.myNode.destroy(function (err, result) {
     if (err) {
-      onComplete(err, result);
-      return;
+      logger.warn('Error destroying node:' + err.message);
+      // Don't stop disconnecting on an error here; we make a good-faith
+      // effort to clean up after ourselves, and let the cleaning system take
+      // care of the rest.
```

Comment:
This is the second layer of the problem that I found after fixing the wire delete issue; same problem existed when trying to destroy the user node.